src/lib/parse_release.c

```diff
@@ -289,6 +289,19 @@ void parse_osinfo_for_bz(map_string_t *osinfo, char** product, char** version)
     *version = NULL;
 }
 
+void parse_osinfo_for_bug_url(map_string_t *osinfo, char** url)
+{
+    const char *os_bug_report_url = get_map_string_item_or_NULL(osinfo, "BUG_REPORT_URL");
+
+    if (os_bug_report_url)
+        *url = xstrdup(os_bug_report_url);
+    else
+        /* unset or something bad happend */
+        *url = NULL;
+
+    return;
+}
+
```

Comment:
Alright then. Since this is a public function we should cover it with unit tests. Please add a new test case to https://github.com/abrt/libreport/blob/master/tests/osinfo.at . Here you can find how the unit tests work: https://github.com/abrt/abrt/blob/master/HACKING.md#how-to-check-your-changes.

Just a side note, the function can be used in `reporter-mantisbt` too.